sdk/helper/consts/consts.go

```diff
@@ -34,4 +34,6 @@ const (
 	ReplicationResolverALPN = "replication_resolver_v1"
 
 	VaultEnableFilePermissionsCheckEnv = "VAULT_ENABLE_FILE_PERMISSIONS_CHECK"
+
+	VaultDisableUserLockout = "VAULT_DISABLE_USER_LOCKOUT"
```

Comment:
I don't see this in use. Was it maybe forgotten or did we choose not to allow this environment variable override?

Reply:
I added this so that we could use this to check for environment variable while working on future tickets (Prevent brute forcing)